### PR TITLE
NOISSUE - Replace repeating code by card gen func

### PR DESCRIPTION
**Pull request by darkodraskovic**
Signed-off-by: Darko Draskovic <darko.draskovic@gmail.com>

### What does this do?

Replaces repeating code in Connection and Message module by a helper func.

### CODE REVIEW

File: ui/README.md [hunk 1]
```diff
@@ -46,10 +46,12 @@ Open the _src/Env.elm_ file and edit the values of the `env` record.
 
 Follow the instructions above to install and run GUI as a native build. In
 _src/Env.elm_ change a `url` field value of the `elm` record to
-`http://localhost:80/` (trailing slash `/` is mandatory). Instead of `make run` you can install `elm-live`
-(https://github.com/wking-io/elm-live) and execute `elm-live src/Main.elm` to
-get a live reload when your `.Elm` files change.
+`http://localhost:80/` (trailing slash `/` is mandatory). Instead of `make run`
+you can install `elm-live` (https://github.com/wking-io/elm-live) and execute
```
Comment (drasko) [inline ui/README.md]: Use Markdown links

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by blokovi — DISMISSED** [review]
LGTM

**Review by drasko — DISMISSED** [review]
LGTM

**Review by blokovi — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**